core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -84,6 +85,7 @@ class KryoSerializer(conf: SparkConf)
   private val avroSchemas = conf.getAvroSchema
   // whether to use unsafe based IO for serialization
   private val useUnsafe = conf.getBoolean("spark.kryo.unsafe", false)
+  private val usePool = conf.getBoolean("spark.kryo.pool", false)
```

Comment:
Yep, I'd leave this undocumented

Reply:
I  had already added this to docs/configuration.md should I remove it?

Reply:
I would not document it. This is just a safety valve. In theory, there's no reason to disable this nor would a caller know why to disable it.

Reply:
Okay, if I'm understanding you correctly I should set the default to true, and remove the documentation. I will go ahead and do that, if I misunderstood let me know. Thanks

Reply:
Yeah I think that's a fine position to take, if we can't think of a reason to disable it other than the theoretical unknown unknown bug out there.

Reply:
Sounds good and done